tests/test_mapping.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import torch
+
+
+class TestGetPeftModel:
+    RELOAD_WARNING_EXPECTED_MATCH = r"You are trying to modify a model .*"
+
+    @pytest.fixture
```

Comment:
In this project, we just import the PEFT functions and classes directly on the root of the test files, so it's not really necessary to use a fixture in this case.